backend/tests/v5/e2e/routes/teamspaces/projects/models/common/groups.test.js

```diff
@@ -0,0 +1,215 @@
+/**
+ *  Copyright (C) 2021 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+const SuperTest = require('supertest');
+const ServiceHelper = require('../../../../../../helper/services');
+const { src } = require('../../../../../../helper/path');
+
+const { templates } = require(`${src}/utils/responseCodes`);
+
+let server;
+let agent;
+
+const testExportGroups = () => {
+	const users = {
+		tsAdmin: ServiceHelper.generateUserCredentials(),
+		noProjectAccess: ServiceHelper.generateUserCredentials(),
+		nobody: ServiceHelper.generateUserCredentials(),
+	};
+
+	const teamspace = ServiceHelper.generateRandomString();
+
+	const project = {
+		id: ServiceHelper.generateUUIDString(),
+		name: ServiceHelper.generateRandomString(),
+	};
+
+	const container = ServiceHelper.generateRandomModel();
+	const containerNoGroups = ServiceHelper.generateRandomModel();
+	const fed = ServiceHelper.generateRandomModel({ isFederation: true });
+	const fedNoGroups = ServiceHelper.generateRandomModel({ isFederation: true });
+
+	const groups = [
```

Comment:
shouldn't we include some federation groups as well ?

Reply:
same as above! 😆